meta-freescale-mbl/recipes-bsp/imx-mkimage/imx-boot_0.2.bbappend

```diff
@@ -22,3 +21,8 @@ SRC_URI += "file://0001-iMX8M-Add-a-FIP-entry-into-the-mkimage-command.patch \
 do_compile_prepend() {
     install -m 0644 ${DEPLOY_DIR_IMAGE}/optee/tee.bin ${BOOT_STAGING}
 }
+
+# Bitbake doesn't reprocess this recipe after cleansstate any of the recipes listed
+# in the do_compile[depends]. The 'nostamp' is a workaround to force this recipe
+# to be processed every time we build an image recipe.
```

Comment:
```suggestion
# to be processed every time we build an image recipe.
# This is needed because we need to ensure that the initial boot image is
# rebuilt for the update payload creation.
# After the imx8 secure boot full port this workaround should not be needed
# anymore.
```